### PR TITLE
Assign FEBI method

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**
Object-oriented not implemented in AEDT API when Hybrid region

### CODE REVIEW

**Comment by Samuelopez-ansys** [comment]
Added Transient composite solution type.

It is missed the default setup. I realized it would fail because the transient setup is different than the frequency one. It will be refactored in a different PR.

File: pyaedt/hfss.py [hunk 1]
```diff
@@ -4561,6 +4565,49 @@ def assign_hybrid_region(self, obj_names, boundary_name="", hybrid_region="SBR+"
...
+
+        References
+        ----------
+
+        >>> oModule.AssignFEBI
+
+        Examples
+        --------
+
+        Create a box and assign a FEBI boundary to it.
```
Comment (PipKat) [inline pyaedt/hfss.py:4592]:
```suggestion
        Create a box and assign an FEBI boundary to it.
```

File: pyaedt/hfss.py [hunk 2]
```diff
@@ -4561,6 +4565,49 @@ def assign_hybrid_region(self, obj_names, boundary_name="", hybrid_region="SBR+"
...
+    @pyaedt_function_handler()
+    def assign_febi(self, obj_names, boundary_name=""):
+        """Assign an FE-BI region to one or more objects.
+
+        Parameters
+        ----------
+        obj_names : str or list or int or :class:`pyaedt.modeler.cad.object3d.Object3d`
+            One or more object names or IDs.
+        boundary_name : str, optional
+            Name of the boundary. The default is ``""``.
```
Comment (PipKat) [inline pyaedt/hfss.py:4577]: Perhaps we should include the bevavior that occurs when ``""`` is used, something like this:
The default is ``""``, in which case a name is automatically assigned.

File: pyaedt/hfss.py [hunk 3]
```diff
@@ -4561,6 +4565,49 @@ def assign_hybrid_region(self, obj_names, boundary_name="", hybrid_region="SBR+"
...
+
+        References
+        ----------
+
+        >>> oModule.AssignFEBI
+
+        Examples
+        --------
+
+        Create a box and assign an FEBI boundary to it.
```
Comment (PipKat) [inline pyaedt/hfss.py:4592]:
```suggestion
        Create a box and assign an FE-BI boundary to it.
```
Check for consistency. It looks like FE-BI is preferred.

File: pyaedt/hfss.py (hunk 2 above)
Comment (Samuelopez-ansys) [inline pyaedt/hfss.py:4577]:
```suggestion
            Name of the boundary. The default is ``""``, in which case a name is automatically assigned.
```

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Pull request merged**